doc/models_library/aeif_cond_alpha.rst

```diff
@@ -92,103 +88,77 @@ Equations
-       w pA = 0pA # Spike-adaptation current
-     end
-     equations:
-       inline V_bounded mV = min(V_m,V_peak) # prevent exponential divergence
-       kernel g_inh = (e / tau_syn_inh) * t * exp(-t / tau_syn_inh)
-       kernel g_exc = (e / tau_syn_exc) * t * exp(-t / tau_syn_exc)
-       # Add inlines to simplify the equation definition of V_m
-       inline exp_arg real = (V_bounded - V_th) / Delta_T
-       inline I_spike pA = g_L * Delta_T * exp(exp_arg)
-       inline I_syn_exc pA = convolve(g_exc,exc_spikes) * (V_bounded - E_exc)
-       inline I_syn_inh pA = convolve(g_inh,inh_spikes) * (V_bounded - E_inh)
-       V_m'=(-g_L * (V_bounded - E_L) + I_spike - I_syn_exc - I_syn_inh - w + I_e + I_stim) / C_m
-       w'=(a * (V_bounded - E_L) - w) / tau_w
-     end
-
-     parameters:
-       # membrane parameters
-       C_m pF = 281.0pF # Membrane Capacitance
-       t_ref ms = 0.0ms # Refractory period
-       V_reset mV = -60.0mV # Reset Potential
-       g_L nS = 30.0nS # Leak Conductance
-       E_L mV = -70.6mV # Leak reversal Potential (aka resting potential)
-       # spike adaptation parameters
-
-       # spike adaptation parameters
-       a nS = 4nS # Subthreshold adaptation
-       b pA = 80.5pA # Spike-triggered adaptation
-       Delta_T mV = 2.0mV # Slope factor
-       tau_w ms = 144.0ms # Adaptation time constant
-       V_th mV = -50.4mV # Threshold Potential
-       V_peak mV = 0mV # Spike detection threshold
-       # synaptic parameters
-
-       # synaptic parameters
-       E_exc mV = 0mV # Excitatory reversal Potential
-       tau_syn_exc ms = 0.2ms # Synaptic Time Constant Excitatory Synapse
-       E_inh mV = -85.0mV # Inhibitory reversal Potential
-       tau_syn_inh ms = 2.0ms # Synaptic Time Constant for Inhibitory Synapse
-       # constant external input current
-
-       # constant external input current
-       I_e pA = 0pA
-     end
-     internals:
-       # Impulse to add to DG_EXC on spike arrival to evoke unit-amplitude conductance excursion
-       PSConInit_E nS/ms = nS * e / tau_syn_exc
-       # Impulse to add to DG_INH on spike arrival to evoke unit-amplitude conductance excursion
-       PSConInit_I nS/ms = nS * e / tau_syn_inh
-       # refractory time in steps
-       RefractoryCounts integer = steps(t_ref)
-       # counts number of tick during the refractory period
-
-       # counts number of tick during the refractory period
-       r integer 
-     end
-     input:
-       inh_spikes nS <-inhibitory spike
-       exc_spikes nS <-excitatory spike
-       I_stim pA <-current
-     end
-
-     output: spike
-
-     update:
-       integrate_odes()
-       if r > 0: # refractory
-         r -= 1 # decrement refractory ticks count
-         V_m = V_reset # clamp potential
-       elif V_m >= V_peak:
-         r = RefractoryCounts
-         V_m = V_reset # clamp potential
-         w += b
-         emit_spike()
-       end
-     end
-
-   end
+       state: # Membrane potential
```

Comment:
This looks messy as the whitespace is not preserved, shall we replace the source code print with a link to the model instead?